cran-comments.md

```diff
@@ -32,6 +32,24 @@ Namespace in Imports field not imported from: 'BH'
 
 Maintainer response: The BH package provides Boost C++ library headers. The BH library is not used in any of the code in /R or /src, but only in the C++ templates that comes with the package (in /inst/cpp). The C++ templates are used when creating a model using the new_ode_model() function, which is then compiled on-the-fly. If the BH package is removed from the Depends field, it won't be installed and hence models cannot be compiled and run. BH cannot be moved to the LinkingTo field.
 
+* CRAN repository db overrides:
+  X-CRAN-Comment: Archived on 2022-01-19 for policy violation.
+
+  On not remving debugging information.
```

Comment:
```suggestion
  On not removing debugging information.
```

Reply:
that typo was in the CRAN output, so have to keep...